edpop-record-ontology.ttl

```diff
@@ -0,0 +1,140 @@
+                     rdfs:domain edpoprec:Record ;
+                     rdfs:range edpoprec:Catalog .
+
+edpoprec:identifier a rdf:Property ;
+                    skos:prefLabel "Identifier"@en ;
+                    skos:description "Unique identifier used by the source catalog" ;
+                    rdfs:domain edpoprec:Record ;
+                    rdfs:range edpoprec:Catalog .
+
+edpoprec:publicURL a rdf:Property ;
+                   skos:prefLabel "Public URL"@en ;
+                   skos:description "Public URL in the source catalog" ;
+                   rdfs:domain edpoprec:Record ;
+                   rdfs:range edpoprec:Catalog .
+
+# Original data: supposed to contain a blank record with all (relevant) original data,
+# rudimentally converted into RDF, so that it will be possible to show this data in the VRE.
+edpoprec:originalData a rdf:Property ;
+                      skos:prefLabel "Original data"@en ;
+		      skos:description "All data from the original catalogue record" ;
+		      rdfs:domain edpoprec:Record .
+
+### PROPERTIES SPECIFIC FOR BIBLIOGRAPHICAL RECORDS
+
+# So far we only define string literals as ranges for these properties, but we could think
+# of also allowing other classes, like our own biographical records for authors.
+
+# The string literals are meant to be the best effort of EDPOP Explorer to give a 
+# somehow normalized, human-readable representation of the property, but they are not
+# thesaurized and certainly not guaranteed to be the same if they are conceptually the same
+# (e.g., Köln or Keulen for the city of Cologne).
+
+# My proposal would be to use these properties that can be extracted from the original record
+# in a deterministic way. That means: only string literals, and also links to other records
+# if they are defined in the original catalogues (such as links to CT Thesaurus).
+# Users will be able to enrich, correct and further normalize this data with annotations.
+# Annotations may also be used, perhaps, for automatic non-deterministic normalizations,
+# such as the place of publication as a link to Wikidata.
+
+edpoprec:title a rdf:property ;
+               skos:prefLabel "Title"@en ;
+               rdfs:domain edpoprec:BibliographicalRecord ;
+               rdfs:range xsd:string .
+
+edpoprec:alternativeTitle a rdf:property ;
+                            skos:prefLabel "Alternative title"@en ;
+                            rdfs:domain edpoprec:BibliographicalRecord ;
+                            rdfs:range xsd:string .
+
+# Or: contributor? And how to define the role of the contributor? Perhaps a separate
+# class is better. The catalogues generally describe the roles of the contributors quite
+# well (author, translator, illustrator, etc.)
+edpoprec:author a rdf:property ;
+                skos:prefLabel "Author"@en ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
```

Comment:
You could define a `ContributionRole` class and a couple of named instances:

```ttl
edpoprec:author a edpoprec:ContributionRole.
edpoprec:translator a edpoprec:ContributionRole.
edpoprec:illustrator a edpoprec:ContributionRole.
edpoprec:printer a edpoprec:ContributionRole.
```

Then, when specifiying the contributors of a record, I can think of at least two possible ways to specify their roles. Both of the following approaches allow for zero or more roles for the same contributor (where zero roles could mean "role not clearly specified").

Option 1, use RDF* to annotate the role(s):

```ttl
vredata:someBookRecord edpoprec:contributor "Tijmen Baarda".
<<vredata:someBookRecord edpoprec:contributor "Tijmen Baarda">> edpoprec:role edpoprec:translator.
```

This would require using Blazegraph in "RDR mode".

Option 2, use a blank node and `rdf:value`:

```ttl
vredata:someBookRecord edpoprec:contributor [
    rdf:value "Tijmen Baarda";
    edpoprec:role edpoprec:translator.
].
```

I used `edpoprec:role` in these examples, but maybe that name should be a bit longer to distinguish it from other potential meaning of the word "role".

Reply:
Great! We will certainly have to make the distinction indeed and most catalogues appear to do this too in a fairly consistent way. I will go for Option 2 for now.

Reply:
We could use the list of 'relator terms' of the Library of Congress: https://id.loc.gov/vocabulary/relators.html. All these terms have IRIs and skos metadata.

The relator terms are part of BIBFRAME, which was designed as a successor to Marc21, but unfortunately it has not yet really replaced Marc21 as most libraries still do new cataloguing in Marc21. But we could, perhaps, use BIBFRAME as a basis for our normalizations.